configure.ac

```diff
@@ -13,6 +13,9 @@ AC_DEFINE(VMA_LIBRARY_MINOR, vma_ver_minor, [VMA Minor Version])
 AC_DEFINE(VMA_LIBRARY_REVISION, vma_ver_revision, [VMA Revision])
 AC_DEFINE(VMA_LIBRARY_RELEASE, vma_ver_release, [VMA Release])
 
+define([vma_git_version], esyscmd([sh -c "touch ./build/current-version; head -1 ./build/current-version  | tr -d '\n'"]))
```

Comment:
Let avoid using additional files as `versioning.sh` and `current-version` and get git version on fly as `define([vma_git_version], esyscmd([sh -c "git rev-parse --short=7 HEAD"]))`.
Having additional single purpose auxiliary files makes source folder dirty.